--- diradmin-1.6.0/applnk/dragonfear-directory_administrator.desktop	2005-01-05 19:53:35.000000000 +0000
+++ directory_administrator.desktop	2005-06-02 00:07:56.585747184 +0000
@@ -1,13 +1,12 @@
 [Desktop Entry]
 Name=Directory administrator
-Name[es]=Administrador de directorios
 Comment=Manages LDAP directory users and groups
 Comment[es]=Permite administrar usuarios y grupos en directorios LDAP
+Comment[pl]=Klient LDAP do zarządzania użytkownikami i grupami
 Exec=directory_administrator
 Icon=directory_administrator/diradminlogo.png
 Terminal=false
 Type=Application
-Categories=Application;SystemSetup;X-Red-Hat-Base;
+Categories=GTK;Network;Database;
 Encoding=UTF-8
-X-Desktop-File-Install-Version=0.3
-
+# vi: encoding=utf-8
