--- directory_administrator-1.3.4/configure.in.orig	Thu Jan 23 09:34:54 2003
+++ directory_administrator-1.3.4/configure.in	Thu Jan 23 09:35:46 2003
@@ -24,7 +24,7 @@
 PACKAGE_PIXMAPS_DIR="/share/pixmaps/${PACKAGE}"
 AC_SUBST(PACKAGE_PIXMAPS_DIR)
 dnl Subst PACKAGE_APPLNK_DIR.
-PACKAGE_APPLNK_DIR="/share/applications"
+PACKAGE_APPLNK_DIR="/share/applnk/Network/Misc"
 SYSTEM_MENU_DIR="/lib/menu"
 
 AC_OUTPUT([
